var/spack/repos/builtin/packages/py-nanobind/package.py

```diff
@@ -23,6 +23,24 @@ class PyNanobind(PythonPackage):
     maintainers("ma595")
 
     version("master", branch="master", submodules=True)
+    version(
+        "1.5.2",
+        tag="v1.5.2",
+        commit="2574b91ba15d6160cbc819eb72da3c885601b0468e0d9eda83fc14d3be996113",
```

Comment:
These commit hashes don't seem right, I'm seeing b0e24d5b0ab0d518317d6b263a257ae72d4d29a2 for v1.5.2

Reply:
Fixed: I used the file checksums :).